src/test/contact-info/edit.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * External dependencies
+ */
+import {
+	getEditorHtml,
+	initializeEditor,
+	addBlock,
+	getBlock,
+	setupCoreBlocks,
+} from 'test/helpers';
+
+/**
+ * Internal dependencies
+ */
+import {
+	registerJetpackBlocks,
+	setupJetpackEditor,
+} from '../../jetpack-editor-setup';
+
+setupCoreBlocks();
+
+beforeAll( () => {
+	// Register Jetpack blocks
+	setupJetpackEditor( {
+		blogId: 1,
+		isJetpackActive: true,
+	} );
+	registerJetpackBlocks( {
+		capabilities: {
+			contactInfoBlock: true,
+		},
+	} );
+} );
+
+describe( 'Contact Info block', () => {
+	it( 'should successfully insert the block into the editor', async () => {
+		const screen = await initializeEditor();
+
+		// Add block
+		await addBlock( screen, 'Contact Info' );
+
+		// Get block
+		const block = await getBlock( screen, 'Contact Info' );
+		expect( block ).toBeVisible();
+
+		expect( getEditorHtml() ).toMatchSnapshot();
```

Comment:
Nitpick: group the assertions together. I recognize you were grouping the usage of `block`, so feel free to ignore this suggestion. 

```suggestion
		const block = await getBlock( screen, 'Contact Info' );

		expect( block ).toBeVisible();
		expect( getEditorHtml() ).toMatchSnapshot();
```

Reply:
Suggestion applied in https://github.com/wordpress-mobile/gutenberg-mobile/pull/6049/commits/c7cf994c24c08ab63e8db29d9c16e82c20763b97.